semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -91,14 +91,25 @@ trait TextDocumentOps { self: SemanticdbOps =>
           private def indexArgNames(mapp: m.Tree, mnames: List[m.Name]): Unit = {
             if (mnames.isEmpty) return
             todo ++= mnames
-            val mstart1 = mapp.tokens
-              .dropWhile(_.is[m.Token.LeftParen])
-              .headOption
-              .map(_.start)
-              .getOrElse(-1)
-            // only add names for the top-level term.apply of a curried function application.
-            if (!margnames.contains(mstart1))
-              margnames(mstart1) = mnames
+            val mappTokens = mapp.tokens.reverse
```

Comment:
Could you paste somewhere in the comment how `mapp.tokens` could look?

Reply:
For mapp `a.tst1(i = 1).tst2(j = 2)` mapp.tokens looks like `|a|.|tst1|(|i| |=| |1|)|.|tst2|(|j| |=| |2|)`. I've added | to show where one token starts and where it ends.